imap_processing/idex/idex_packet_parser.py

```diff
@@ -125,6 +142,11 @@ def __init__(self, header_packet):
             self.high_sample_trigger_time,
         ) = self._calc_sample_trigger_times(header_packet)
 
+        self.trigger_values_dict, self.trigger_notes_dict = self._get_trigger_dicts(
+            header_packet
+        )
+        logging.debug(f"{self.trigger_values_dict}")  # Log values here in case of error
```

Comment:
```suggestion
        logging.debug(f"trigger_values_dict:\n{self.trigger_values_dict}")  # Log values here in case of error
```
Something to indicate what this is logging otherwise it might be tough to tie the dictionary to the area of interest.